requirements/build.txt

```diff
@@ -7,4 +7,4 @@ sphinx-rtd-theme
 sphinx-autobuild
 pex
 wheel
-setuptools<20.11,>=2.2          # needed to get pex working
+setuptools==38.2.4          # needed to get pex working
```

Comment:
looks like this will break things

Reply:
Will configure it to be ignored for now. It's not important as it's not used in runtime, just for building the Pex IIRC.